dpctl/tensor/_copy_utils.py

```diff
@@ -169,12 +180,16 @@ def asnumpy(usm_ary):
     asnumpy(usm_ary)
 
     Copies content of :class:`dpctl.tensor.usm_ndarray` instance `usm_ary`
-    into `numpy.ndarray` instance of the same shape and same data type.
+    into :class:`numpy.ndarray` instance of the same shape and same data
+    type.
 
     Args:
-        usm_ary: An instance of :class:`dpctl.tensor.usm_ndarray`
+        usm_ary (usm_ndarray): An instance of
+            :class:`dpctl.tensor.usm_ndarray`
     Returns:
-        An instance of `numpy.ndarray` populated with content of `usm_ary`.
+        numpy.ndarray:
+            An instance of :class:``numpy.ndarray`` populated with content
```

Comment:
```suggestion
            An instance of :class:`numpy.ndarray` populated with content
```
The ``[...]`` does not seem to render correctly.